services/QuillConnect/app/libs/grading/rematching.ts

```diff
@@ -99,7 +99,7 @@ export function paginatedNonHumanResponses(matcher, matcherFields, qid, page, ca
 
   return request(
     {
-      uri: `https://cms.quill.org/questions/${qid}/responses/search`,
+      uri: `${process.env.QUILL_CMS}/questions/${qid}/responses/search`,
```

Comment:
You probably ran into this while testing locally, but I do approve of you taking the initiative to commit these changes to parameterize the URL host.  Good maintenance habit.

Reply:
Thanks! It probably should have been a separate PR...